src/types.db

```diff
@@ -232,6 +232,8 @@ spam_score              value:GAUGE:U:U
 spl                     value:GAUGE:U:U
 swap                    value:GAUGE:0:1099511627776
 swap_io                 value:DERIVE:0:U
+synproxy_connections    value:DERIVE:0:U
```

Comment:
Plugin-specific types are very discouraged. I recommend to use the existing `connections` type and rename the other type to simply `cookies`.